types/evse_manager.yaml

```diff
@@ -558,3 +558,24 @@ types:
       - IncompatibleEnergyTransfer
       - ServiceRenegotiationFailed
       - NoHlc
+  OperationModeEnum:
```

Comment:
Same here, I would drop this for now, until we know if this belongs rather to OCPP, EnergyManager or the EvseManager